packages/venia-ui/lib/components/Link/link.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { bool } from 'prop-types';
+import useLink from '@magento/peregrine/lib/talons/Link/useLink';
+
+const removeProps = (props, exclude) => {
+    return Object.fromEntries(
+        Object.entries(props).filter(([name]) => !exclude.includes(name))
+    );
+};
+
+const Link = props => {
+    const talonProps = useLink(props);
+    const { ref } = talonProps;
+    const propsForBase = removeProps(props, ['prefetchType', 'innerRef']);
+
+    return <RouterLink {...propsForBase} innerRef={ref} />;
+};
+
+Link.defaultProps = {
+    prefetchType: false
+};
+
+Link.propTypes = {
+    prefetchType: bool
```

Comment:
I don't quite follow this prop. The name doesn't indicate that the type is going to be a boolean. I feel like this will be a source of a fair bit of confusion for devs. In `useLink` the destructured prop gets renamed to something that _does_ indicate a boolean type, but a developer won't see that, the exposed API for this component indicates `prefetchType`